sphinxgallery/backreferences.py

```diff
@@ -124,29 +124,32 @@ def scan_used_functions(example_file, gallery_conf):
     return backrefs
 
 
-def _thumbnail_div(full_dir, fname, snippet):
-    """Generates RST to place a thumbnail in a gallery"""
-    thumb = os.path.join(full_dir, 'images', 'thumb',
-                         'sphx_glr_%s_thumb.png' % fname[:-3])
-    ref_name = os.path.join(full_dir, fname).replace(os.path.sep, '_')
-    if ref_name.startswith('._'):
-        ref_name = ref_name[2:]
-
-    out = """
+THUMBNAIL_TEMPLATE = """
 .. raw:: html
 
-    <div class="sphx-glr-thumbContainer" tooltip="{}">
+    <div class="sphx-glr-thumbContainer" tooltip="{snippet}">
 
-.. figure:: /{}
+.. figure:: /{thumbnail}
 
-    :ref:`example_{}`
+    :ref:`example_{filename}`
```

Comment:
Only minor comment is that it is not really a filename (slashes have been replaced by underscores) but I don't really have a better naming suggestion.

Other than that feel free to merge.